packages/shared/sdk-client/src/LDClientImpl.ts

```diff
@@ -354,15 +356,23 @@ export default class LDClientImpl implements LDClient {
     this.logger.debug(`Identifying ${JSON.stringify(this.checkedContext)}`);
 
     const loadedFromCache = await this.flagManager.loadCached(this.checkedContext);
-    if (loadedFromCache) {
+    if (loadedFromCache && !waitForNetworkResults) {
+      this.logger.debug('Identify completing with cached flags');
       identifyResolve();
     }
+    if (loadedFromCache && waitForNetworkResults) {
+      this.logger.debug(
+        'Identify - Flags loaded from cache, but identify was requested with "waitForNetworkResults"',
+      );
+    }
 
     if (this.isOffline()) {
       if (loadedFromCache) {
-        this.logger.debug('Offline identify using storage flags.');
+        this.logger.debug('Offline identify - using cached flags.');
       } else {
-        this.logger.debug('Offline identify no storage. Defaults will be used.');
+        this.logger.debug(
+          'Offline identify - no cached flags, using defaults or already loaded flags.',
```

Comment:
```suggestion
          'Identify - Currently offline with no cached flags for provided context.  Maintaining current flags. This may result in default flag evaluations if no flags have been loaded before.'
```

Is the trailing comma intentional?

Reply:
Our linting requires training coma for parameters that are not on one line. The linter also forces this line to be wrapped as such.